readme.md

```diff
@@ -7,6 +7,31 @@ This repository contains the following:
 - **Microsoft Azure Provisioning device SDK for C#** to provision devices to Azure IoT Hub with .NET
 - **Microsoft Azure Provisioning service SDK for C#** to manage your Provisioning service instance from a back-end .NET application
 
+### Build status
+
+Due to security considerations, build logs are not publicly available.
+
+| Service Environment                                                   | Status |
+| ---                                                                   | ---    |
+| [Master](https://github.com/Azure/azure-iot-sdk-csharp/tree/master)   | [![Build Status](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_apis/build/status/csharp/csharp-westus1?branchName=master)](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_build?definitionId=44&_a=summary&repositoryFilter=9&branchFilter=23)|
+| [Preview](https://github.com/Azure/azure-iot-sdk-csharp/tree/preview) | [![Build Status](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_apis/build/status/csharp/csharp-canary1?branchName=preview)](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_build?definitionId=43&_a=summary&repositoryFilter=9&branchFilter=72)|
+
+### Recommended NuGet packages
+
+| Package Name                                          | Release Version                                           | Pre-release Version   |
+| ---                                                   | ---                                                       | ---                   |
+| Microsoft.Azure.Devices.Client                        | [![NuGet][iothub-device-release]][iothub-device-nuget]    | [![NuGet][iothub-device-prerelease]][iothub-device-nuget]     |
```

Comment:
Should we also add the license badge for these packages?

Reply:
Good idea, I can add one right next to the `Microsoft Azure IoT SDK for .NET` header, what do you think?

Reply:
That should be good

Reply:
The license badge is hitting some issues with GitHub's rate limits. The workaround is to install a "Shields GitHub application", which requires some additional investigation. I'll hold off on adding this change in this PR.